js/server_config.js

```diff
@@ -7,4 +7,4 @@ configuration for the api
 Author:
 Nilusink
 */
-const ADDRESS = "ws://192.168.204.207:9999"
\ No newline at end of file
+const ADDRESS = "ws://192.168.4.42:9999"
```

Comment:
Don't commit this file. The ADDRESS variable should be dynamically set to the servers host address.